ghp.go

```diff
@@ -55,3 +56,40 @@ func main() {
 	http.HandleFunc("/", handler)
 	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", port), nil))
 }
+
+func renderFile(w http.ResponseWriter, r *http.Request, filename string) {
+	nameSlice := strings.Split(filename, ".")
```

Comment:
Consider using
```suggestion
	ext:= path.Ext(filename)
```
and adding `"path"` to the import statement at the top.